pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -249,6 +249,7 @@ func (c *Reconciler) durationAndCountMetrics(ctx context.Context, pr *v1beta1.Pi
 		newPr, err := c.pipelineRunLister.PipelineRuns(pr.Namespace).Get(pr.Name)
 		if err != nil {
```

Comment:
I’d prefer logging differently for not found vs other errors, using `errors.IsNotFound(err)` like at https://github.com/tektoncd/pipeline/blob/9d60d0adf41e74f78029a0eaab73140fa4f7206a/pkg/status/status.go#L38

Reply:
Thanks. I changed for not found.